services/core/shared/core/compat/sdk_v5/schema/accounts.js

```diff
@@ -21,8 +21,8 @@ module.exports = {
 		isDelegate: { type: 'boolean' },
 		balance: { type: 'bigInteger' },
 		username: { type: 'string' },
-		rewards: { type: 'bigInteger' },
-		producedBlocks: { type: 'integer' },
+		rewards: { type: 'bigInteger', default: true, defaultValue: 0 },
+		producedBlocks: { type: 'integer', default: true, defaultValue: 0 },
```

Comment:
We can remove the `default` params below and update `mysql.js` as below.
This would help us keep the schema pollution-free (i.e no non-knex related configs).

```suggestion
		rewards: { type: 'bigInteger', defaultValue: 0 },
		producedBlocks: { type: 'integer', defaultValue: 0 },
```